Tests/UITests/ResponseStubs/slide-up-trigger-invalid-url.json

```diff
@@ -0,0 +1,64 @@
+{
+    "currentPingMillis": 0,
+    "nextPingMillis": 3600000,
+    "data": [
+        {
+            "campaignData": {
+                "campaignId": "slide-up-trigger",

```

Comment:
please use "slide-up-trigger-invalid-url" as campaignId (just in case)

Reply:
Updated here [e9fcb20](https://github.com/rakutentech/ios-inappmessaging/pull/269/commits/e9fcb2087c423e15b3dcd70d4445dd2b56505760).